app/setup.py

```diff
@@ -64,8 +64,12 @@ def create_app(setting_overrides=None):  # noqa: C901  pylint: disable=too-compl
 
     application.eq = {}
 
-    secrets = yaml.safe_load(open(application.config['EQ_SECRETS_FILE']))
-    keys = yaml.safe_load(open(application.config['EQ_KEYS_FILE']))
+    with open(application.config['EQ_SECRETS_FILE']) as secrets_file:
+        secrets = yaml.safe_load(secrets_file)
+
```

Comment:
Should you not be calling the close method the file like this i.e. secrets.close()? as best practice

Reply:
http://effbot.org/zone/python-with-statement.htm

Reply:
ahhh i was looking at: http://www.tutorialspoint.com/python/python_files_io.htm but forgot bout the with part